### PR TITLE
Clean up and fix build

### PR DESCRIPTION
**Pull request by philipktlin**
1. Remove to publish IotEdgeQuickStart and LeafDevice to old folder; all published files should be found in x64 and arm32v7 folder.
2. Update TemperatureFilter project to copy all docker files.
3. Update docker build to use base_registry build arg.

### CODE REVIEW

**Comment by philipktlin** [comment]
First I found that <Content> is more friendly, which will automatically copy all docker files, even newly added ones.  Second this is the same approach we did for all other modules except TemperatureFilter.

File: edge-modules/TemperatureFilter/TemperatureFilter.csproj [hunk 1]
```diff
@@ -19,6 +19,10 @@
     <OutputPath>bin\CodeCoverage</OutputPath>
     <DefineConstants>DEBUG;TRACE</DefineConstants>
   </PropertyGroup>
+  
+  <ItemGroup>
+    <Content Include="docker*/**/*.*" CopyToPublishDirectory="Always" />
```
Comment (varunpuranik) [inline edge-modules/TemperatureFilter/TemperatureFilter.csproj]: Is *.* correct? Is this not a pattern?

File: edge-modules/SimulatedTemperatureSensor/docker/windows/arm32v7/Dockerfile [hunk 2]
```diff
@@ -1,5 +1,6 @@
 ARG base_tag=1.0.0-windows-arm32v7
-FROM azureiotedge/azureiotedge-module-base:${base_tag}
+ARG base_registry
+FROM ${base_registry}/azureiotedge/azureiotedge-module-base:${base_tag}
```
Comment (varunpuranik) [inline edge-modules/SimulatedTemperatureSensor/docker/windows/arm32v7/Dockerfile]: Is using the public base registry not by design?

Comment (philipktlin) [reply edge-modules/SimulatedTemperatureSensor/docker/windows/arm32v7/Dockerfile]: I will confirm with Damon next Monday.

File: edge-modules/TemperatureFilter/TemperatureFilter.csproj (hunk 1 above)
Comment (philipktlin) [reply edge-modules/TemperatureFilter/TemperatureFilter.csproj]: I tested and it is used in other existing project except this one.

**Pull request merged**